Java/other/Transpose square matrix.java

```diff
@@ -0,0 +1,65 @@
+//Transpose of a matrix is obtained by changing rows to columns and columns to rows. In other words, transpose of A[][] is obtained by changing A[i][j] to A[j][i].
+
```

Comment:
- Use multiline comments
- Make it short and understandable
- Add a bit more description